tool/gravity/cli/ops.go

```diff
@@ -106,6 +107,13 @@ func uploadUpdate(env *localenv.LocalEnvironment, opsURL string) error {
 		return trace.Wrap(err)
 	}
 
+	if cluster.State == ops.SiteStateDegraded {
+		return trace.BadParameter("The cluster is in degraded state so " +
+			"uploading new applications is prohibited. Please check " +
+			"gravity status output and correct the situation before " +
+			"attempting again.")
```

Comment:
```
It is not possible to upload new applications while the cluster is not healthy.
Please make sure the cluster is healthy before reattempting the update.
```
I think `prohibited` is a too strong here.

Reply:
I think "not possible" is actually stronger than "prohibited" (and not entirely true: it is possible but not allowed). Also my wording hints them at the specific action ("check gravity status") rather than just saying "make sure cluster is healthy".